src/pages/workspace/WorkspaceNewRoomPage.tsx

```diff
@@ -164,7 +163,7 @@ function WorkspaceNewRoomPage() {
             } else if (ValidationUtils.isReservedRoomName(values.roomName)) {
                 // Certain names are reserved for default rooms and should not be used for policy rooms.
                 ErrorUtils.addErrorMessage(errors, 'roomName', translate('newRoomPage.roomNameReservedError', {reservedName: values.roomName}));
-            } else if (ValidationUtils.isExistingRoomName(values.roomName, reports, values.policyID ?? '-1')) {
+            } else if (ValidationUtils.isExistingRoomName(values.roomName, reports, values.policyID)) {
```

Comment:
Why have you made this change? I'd prefer to keep unrelated changes to a minimum here.